### PR TITLE
hoc2015: start blocks.

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

File: apps/src/studio/levels.js [hunk 1]
```diff
@@ -1750,14 +1750,14 @@ levels.js_hoc2015_event_two_items = {
...
   "codeFunctions": {
     "moveRight": null,
     "moveLeft": null,
-    "moveUp": null,
-    "moveDown": null,
+    "whenLeft": null,
+    "whenRight": null
   },
-  "startBlocks": "",
+  "startBlocks": "function whenLeft() {\n  \n}\nfunction whenRight() {\n  \n}",
```
Comment (Bjvanminnen) [inline apps/src/studio/levels.js:1760]: Are you generating these manually or via local levelbuilder?

**Comment by Bjvanminnen** [comment]
lgtm

File: apps/src/studio/levels.js (hunk 1 above)
Comment (breville) [inline apps/src/studio/levels.js:1760]: manually, direct into levels.js

Comment (Bjvanminnen) [inline apps/src/studio/levels.js:1760]: Might make it easier for you to parse/maintain if you stick line breaks in there (current approach is also fine if it works for you), i.e.

```
"startBlocks": "" +
  "function whenLeft() {\n" +
  "  \n" +
  "}\n"
  "function whenRight() {\n" +
  "  \n" +
  "}\n"
```

or

```
"startBlocks": [
  "function whenLeft() {",
  "  ",
  "}",
  "function whenRight() {",
  "  ",
  "}"].join('\n')
```

Comment (breville) [inline apps/src/studio/levels.js:1760]: nice, i'll do one of these shortly!

**Pull request merged**